Calculate/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Linq.Expressions;
+using System.Runtime.CompilerServices;
+using Microsoft.VisualBasic;
+
+namespace Calculate;
+
+public class Program
+{
+    public Func<string?> ReadLine { get; init; } = () => { return Console.ReadLine(); };
+    public Action<string> WriteLine { get; init; } = (value) => { Console.WriteLine(value); };
+
+    public Program()
+    {
+    }
+
+    public Program(Func<string> readLine, Action<string> writeLine)
+    {
+        ReadLine = readLine;
+        WriteLine = writeLine;
+    }
```

Comment:
I'm curious about the intention here as the instructions mention "Set the default behavior for the WriteLine and ReadLine properties to invoke **System.Console** versions of the methods". This makes me think I could use something other than the system defined versions.